pkg/odo/cli/service/create.go

```diff
@@ -212,6 +223,40 @@ func (o *ServiceCreateOptions) Validate() (err error) {
 
 	// we want to find an Operator only if something's passed to the crd flag on CLI
 	if experimental.IsExperimentalModeEnabled() {
+		// if the user wants to create service from a file, we check for
+		// existence of file and validate if the requested operator and CR
+		// exist on the cluster
+		if o.fromFile != "" {
+			if _, err := os.Stat(o.fromFile); err != nil {
+				return errors.Wrap(err, "unable to find specified file")
+			}
+
+			// Parse the file to find Operator and CR info
+			fileContents, err := ioutil.ReadFile(o.fromFile)
+			if err != nil {
+				return err
+			}
+			// var jsonCR map[string]interface{}
+			err = yaml.Unmarshal(fileContents, &o.CustomResourceDefinition)
+			if err != nil {
+				return err
+			}
+
+			// Check if the operator and the CR exist on cluster
+			o.CustomResource = o.CustomResourceDefinition["kind"].(string)
+			csvs, err := o.KClient.GetClusterServiceVersionList()
```

Comment:
this error needs to be handled https://travis-ci.com/github/openshift/odo/jobs/321117025#L528